env.js

```diff
@@ -16,6 +16,8 @@ module.exports = function() {
     docLimit: safeNum(process.env.COUCH2PG_DOC_LIMIT),
     changesLimit: safeNum(process.env.COUCH2PG_CHANGES_LIMIT),
     retryCount: safeNum(process.env.COUCH2PG_RETRY_COUNT),
-    sleepMins: (safeNum(process.env.COUCH2PG_SLEEP_MINS) || 60)
+    sleepMins: (safeNum(process.env.COUCH2PG_SLEEP_MINS) || 60),
+    couchdbUsersMeta: process.env.COUCH2PG_USERS_META || false,
```

Comment:
I think we should remove this environment variable and turn it on for everyone.

Reply:
The ticket states that it should be an opt-in feature by default rather than it being on for everyone.

Reply:
@michaelkohn and I just discussed this and we've both agreed it should be on for everyone. Sorry for the confusion in the issue!

Reply:
No worries. I have made the change. Please have another look when you can.